bin/development-server.js

```diff
@@ -55,6 +56,25 @@ app.get("/babel", function(req, res) {
   res.sendFile(path.join(__dirname, "../babel.html"));
 });
 
+app.get("/chrome-tabs", function(req, res) {
+  const tabRequest = http.get('http://localhost:9222/json/list', (response) => {
```

Comment:
the development server makes the request for tabs from chrome because this endpoint does not support jsonp.

This approach raises an interesting question as to whether we ship the a standalone app with a local server. At the moment, I want to reserve judgement and this was a path forward. I think we could make a chromium contribution for jsonp fairly easily too.